rust/Cargo.lock.in

```diff
@@ -89,8 +89,8 @@ version = "0.4.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "726535892e8eae7e70657b4c8ea93d26b8553afb1ce617caee529ef96d7dee6c"
 dependencies = [
- "proc-macro2 1.0.69",
- "quote 1.0.33",
+ "proc-macro2 1.0.75",
```

Comment:
this appears to update more than the commit message states. If I update just the test-case crate in the Cargo.toml.in and rebuild, I get a smaller diff. How was this updated?

Reply:
`make update-lock` in rust directory

Looking at the Makefile, I see how to make the small update to Cargo.lock.in

Reply:
I'm fine either way, as long as the message reflects what has been done.

Reply:
I thought it did

Reply:
Now I know and I will try to remember